01-bootsector-barebones/README.md

```diff
@@ -1,19 +1,50 @@
-*Concepts you may want to Google beforehand: assembler, BIOS*
+*Concepts you may want to Google beforehand: assembler, BIOS, UEFI, coreboot*
 
 **Goal: Create a file which the BIOS interprets as a bootable disk**
 
 This is very exciting, we're going to create our own boot sector!
 
+Introduction
+------------
+
+After turning power on, boot firmware initializes hardware and run whatever
```

Comment:
typo: After turning power on, boot firmware initializes hardware and **_runs_** whatever